charts/supaglue/templates/salesforce-pub-sub-deployment.yaml

```diff
@@ -0,0 +1,106 @@
+{{- if $.Values.salesforcePubSub.enabled }}
```

Comment:
Can you leave a comment this is experimental, alpha, or for salesforce pubsub only

Reply:
The file name is `salesforce-pub-sub-deployment.yaml`. Do we not think that is sufficient to call it out as for salesforce pub sub?